aws/sdk/examples/sagemaker-list-training-jobs/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use chrono::{DateTime, Utc};
+
+#[tokio::main]
+async fn main() -> Result<(), sagemaker::Error> {
+    let client = sagemaker::Client::from_env();
+    let job_details = client.list_training_jobs().send().await?;
+
+    println!("Job Name\tCreation DateTime\tDuration\tStatus");
+    for j in job_details.training_job_summaries.unwrap_or_default() {
+        let name = j.training_job_name.as_deref().unwrap_or_default();
+        let creation_time_secs = j.creation_time.unwrap().epoch_seconds();
```

Comment:
```suggestion
        let creation_time_secs = j.creation_time.unwrap().to_chrono();
```

Then you don't need to convert it to chrono later

Reply:
I saw that, but to_chrono is marked private, hence the conversion. It's used locally in the module in the fmt method.